src/test/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertificateOrderConfirmationMapperTest.java

```diff
@@ -0,0 +1,183 @@
+ */
+@ExtendWith(SpringExtension.class)
+@SpringJUnitConfig(OrderDataToCertificateOrderConfirmationMapperTest.Config.class)
+public class OrderDataToCertificateOrderConfirmationMapperTest {
+
+    private static final String[] FULL_CERTIFICATE_INCLUDES = new String[]{
+            "Statement of good standing",
+            "Registered office address",
+            "Directors",
+            "Secretaries",
+            "Company objects"
+    };
+
+    private static final String[] CERTIFICATE_INCLUDES_WITHOUT_DIRECTOR_AND_SECRETARY_DETAILS = new String[]{
+            "Statement of good standing",
+            "Registered office address",
+            "Company objects"
+    };
+
+    private static final LocalTime AM = LocalTime.of(7, 30, 15);
+    private static final LocalTime PM = LocalTime.of(15, 30, 15);
+    private static final LocalDate DATE = LocalDate.of(2020, 6, 4);
+    private static final LocalDateTime MORNING_DATE_TIME = LocalDateTime.of(DATE, AM);
+    private static final LocalDateTime AFTERNOON_DATE_TIME = LocalDateTime.of(DATE, PM);
+
+    private static final String EXPECTED_AM_DATE_TIME_RENDERING = "04 June 2020 at 07:30";
+    private static final String EXPECTED_PM_DATE_TIME_RENDERING = "04 June 2020 at 15:30";
+
+    @Configuration
+    @ComponentScan(basePackageClasses = {OrderDataToCertificateOrderConfirmationMapperTest.class})
+    static class Config {}
+
+    @Autowired
+    private OrderDataToCertificateOrderConfirmationMapper mapperUnderTest;
+
+    /**
+     * Implements {@link OrderDataToCertificateOrderConfirmationMapper} to facilitate the testing of its default
+     * methods.
+     */
+    static class TestOrderDataToCertificateOrderConfirmationMapper implements
+            OrderDataToCertificateOrderConfirmationMapper {
+        @Override
+        public CertificateOrderConfirmation orderToConfirmation(OrderData order) {
+            return null; // Implemented only to satisfy requirement of an interface implementation
+        }
+    }
+
+    @Test
+    void orderToConfirmationBehavesAsExpected() {
+
+        // Given
+        final OrderData order = new OrderData();
+        order.setReference("ORD-108815-904831");
+        order.setPaymentReference("orderable_item_ORD-108815-904831");
+
+        final ActionedBy orderedBy = new ActionedBy();
+        orderedBy.setEmail("demo@ch.gov.uk");
+        order.setOrderedBy(orderedBy);
+
+        final DeliveryDetails delivery = new DeliveryDetails();
+        delivery.setForename("Jenny");
+        delivery.setSurname("Wilson");
+        delivery.setAddressLine1("Kemp House Capital Office");
+        delivery.setAddressLine2("LTD");
+        delivery.setLocality("Kemp House");
+        delivery.setPremises("152-160 City Road");
+        delivery.setRegion("London");
+        delivery.setPostalCode("EC1V 2NX");
+
+        order.setDeliveryDetails(delivery);
+        final Item item = new Item();
+        item.setCompanyName("THE GIRLS' DAY SCHOOL TRUST");
```

Comment:
Avoid using real company names/numbers in tests as repo is public

Reply:
God point. Will make changes here.